radix-engine-interface/src/api/api.rs

```diff
@@ -57,6 +57,8 @@ pub trait SysInvokableNative<E>:
     + SysNativeInvokable<AuthZoneClearInvocation, E>
     + SysNativeInvokable<AuthZoneDrainInvocation, E>
     + SysNativeInvokable<AccessRulesAddAccessCheckInvocation, E>
+    + SysNativeInvokable<ComponentSetRoyaltyConfigInvocation, E>
+    + SysNativeInvokable<PackageSetRoyaltyConfigInvocation, E>
```

Comment:
I think these two should be merged into a single invocation as these should be invocations related to royalties and not Component/Package.

Reply:
As discussed we can keep these two as separate invocations for now and reassess later as we gather more evidence/info on the right abstraction